### ISSUE TITLE
Third Meeting

### ISSUE THREAD

**Issue by youngfoo19941** [youngfoo19941/New-Bus-Ticketing-System#3]
Okay guys, now we will start the third sprint planning meeting.

**Comment by YitTuckWeng** [comment]
This is sprint for?

**Comment by youngfoo19941** [comment]
Last week we have discuss the solution of the problem, so you all solve your problem already, my problem is already solve

**Comment by teychoonwei** [comment]
my sql command relly got problem, i also solve the problem already

**Comment by YitTuckWeng** [comment]
Hey guys I have done my system already.

**Comment by freddywg1125** [comment]
Actually when i do retrieve data also meet the problem but I follow zuhui suggestion then solved! Now my system is almost done.

**Comment by teychoonwei** [comment]
The purpose for this sprint planning meeting is want to know all of you is done your system or not,and we need combine together. and i would like to ask you all got any addition method to improve our system

**Comment by youngfoo19941** [comment]
The purpose for this sprint planning meeting is want to know all of you is done your system or not,and we need combine together. and i would like to ask you all got any addition to improve our system

**Comment by teychoonwei** [comment]
i thing we must have a login page and need to set the permission such as limit the clerk cannot edit the staff information

**Comment by NgZuhui** [comment]
i totally agree. and we also can put some validation such as ic no number can not more  than or less than 12 digit

**Comment by freddywg1125** [comment]
@youngfoo19941 I need some time to done my system then only can combine. @NgZuhui I agree with you, this is such a good idea.

**Comment by YitTuckWeng** [comment]
yeah good idea man

**Comment by youngfoo19941** [comment]
totally agree with u @teychoonwei and @NgZuhui . we also need make the ui consistency, so user can easily to use and make user friendly

**Comment by youngfoo19941** [comment]
@freddywg1125 ok.

**Comment by youngfoo19941** [comment]
so if dun have any suggestion, we will end this discussion and remember to improve your system. that all

**Comment by YitTuckWeng** [comment]
ok

**Comment by freddywg1125** [comment]
okyyy
